skt/__init__.py

```diff
@@ -405,11 +405,19 @@ def merge_patchwork_patch(self, uri):
                           patchinfo.get("name").replace(',', ';')))
 
     def merge_patch_file(self, path):
+        if not os.path.isfile(path):
+            raise Exception("Patch %s not found" % os.path.basename(path))
```

Comment:
It would be less confusing to print out the original `path` passed, rather then only the file name. Imagine that two patches have the same file name, but are stored in different directories. It wouldn't be clear which one the exception is about.

Reply:
Oh I see. That makes sense!